tools/eksDistroBuildToolingOpsTools/pkg/constants/constants.go

```diff
@@ -4,4 +4,7 @@ const (
 	Aws                        = "aws"
 	EksdBuildTooling           = "eks-distro-build-tooling"
 	EksGoSupportedVersionsPath = "projects/golang/go/MAINTAINED_EOL_VERSIONS"
+
+	Golang = "golang"
```

Comment:
should we call these `GolangOrgName` and `GolangRepoName` or something more descriptive?